examples/server/server.c

```diff
@@ -45,7 +45,36 @@
 static const char *wolfsentry_config_path = NULL;
 #endif
 #endif /* WOLFSSL_WOLFSENTRY_HOOKS */
-
+#if defined(WOLFSSL_TLS13) && ( \
+       defined(HAVE_ECC) \
+    || defined(HAVE_CURVE25519) \
+    || defined(HAVE_CURVE448) \
+    || defined(HAVE_FFDHE_2048))
+    #define CAN_FORCE_CURVE
+#endif
+#if defined(CAN_FORCE_CURVE) && defined(HAVE_ECC)
+struct group_info {
+    word16 group;
+    const char *name;
+};
+static struct group_info group_id_to_text[] = {
```

Comment:
```

18 [1;38;5;243m--enable-leantls --enable-session-ticket --enable-sni --enable-opensslextra[0m
--
1 examples/server/server.c:60:26: error: ‘group_id_to_text’ defined but not used [-Werror=unused-variable]

18 [[1;38;5;243m--enable-leantls --enable-session-ticket --enable-sni --enable-opensslextra[0m](x-webdoc://055084A3-0FA9-457B-9730-FEA4EEB3252B/log_content.html#INFO18)
1 [examples/server/server.c:60:26: error: ‘group_id_to_text’ defined but not used [-Werror=unused-variable]](x-webdoc://055084A3-0FA9-457B-9730-FEA4EEB3252B/log_content.html#ERROR1)```

Reply:
```

[1;38;5;243m--enable-tls13 CFLAGS="-DNO_WOLFSSL_SERVER"[0m
--
1 examples/server/server.c:61:26: error: ‘group_id_to_text’ defined but not used [-Werror=unused-variable]

[[1;38;5;243m--enable-tls13 CFLAGS="-DNO_WOLFSSL_SERVER"[0m](x-webdoc://94989638-04FE-4734-B106-0E13544294E8/log_content.html#INFO59)
1 [examples/server/server.c:61:26: error: ‘group_id_to_text’ defined but not used [-Werror=unused-variable]](x-webdoc://94989638-04FE-4734-B106-0E13544294E8/log_content.html#ERROR1)```

Reply:
Both of these passed locally. Odd they failed in GitHub CI.

Reply:
Ok seems these are passing now. 24 out of 24.

Reply:
Whoops, wrong account. 24 out of 24.